packages/terra-content-container/tests/nightwatch/DefaultContentContainer.jsx

```diff
@@ -3,11 +3,12 @@ import React from 'react';
 import ContentContainer from '../../lib/ContentContainer';
 
 const header = <span>header</span>;
+const footer = <span>footer</span>;
 const content = <span>content</span>;
 
 const container = () => (
   <div style={{ height: '200px', overflow: 'auto' }}>
```

Comment:
Found a fix - removing the `overflow: 'auto'` inline style on this outermost div reverts overflow to the default, `visible`, and stops the focus shadow from clipping.